packages/launchpad/src/setup/EnvironmentSetup.vue

```diff
@@ -1,12 +1,52 @@
 <template>
+  <Alert
+    v-model="isAlertOpen"
+    :icon="ErrorOutlineIcon"
+    class="mx-auto my-24px max-w-640px"
+    status="warning"
+    :title="t('setupPage.projectSetup.communityFrameworkDefinitionProblem')"
+    dismissible
+  >
+    <p>
+      {{ t('setupPage.projectSetup.communityDependenciesCouldNotBeParsed', erroredFrameworks.length) }}
+    </p>
+    <ul class="list-disc my-12px ml-36px">
+      <li
+        v-for="framework in erroredFrameworks"
+        :key="framework.path as string"
+      >
+        <ExternalLink
+          data-cy="errored-framework-path"
+          :href="`file://${framework.path}`"
```

Comment:
Should this use the `OpenFileInIDE` component? Or is that not an option here?

Reply:
Thought about that switch, I decided against it since this was just about to merge before I took it back over to address some design issues. I didn't see a need to revisit this solution at the moment, I think getting the error displayed is the most important thing.